compute/instances/concepts.mdx

```diff
@@ -118,7 +118,7 @@ Creating an image allows you to make a complete backup of your Instance. One ima
 
 ## Instance
 
-An Instance is a computing unit, either virtual or physical, that provides resources to run your applications on. Currently Scaleway offers the following Instance types: [General Purpose](#general-purpose-instance), [Development](#development-instance), [GPU](#gpu-instance), [Stardust](#stardust-instance) and [Enterprise](#enterprise-instance).
+An Instance is a computing unit, either virtual or physical, that provides resources to run your applications on. Currently Scaleway offers the following Instance types: [General Purpose](#general-purpose-instance), [Development](#development-instance), [GPU](#gpu-instance), [Stardust](#stardust-instance) and [Enterprise](#enterprise-instance). For more information about the different ranges of Instance, see our [dedicated documentation](/compute/instances/reference-content/choosing-instance-type/).
```

Comment:
We are renaming the Enterprise, General Purpose, Development and Stardust ranges => to be updated

Reply:
Do you mean this should be modified to say we offer Learning, Cost-Optimized, Production-Optimized and GPU Instances?